src/SAWScript/Interpreter.hs

```diff
@@ -2744,7 +2744,20 @@ primitives = Map.fromList
     , "`llvm_points_to`). The Term is the tuple consisting of the"
     , "output parameters of the LLVM function: the return parameter, then"
     , "the parameters passed by reference (in the order given by"
-    , "`llvm_points_to`). For more flexibility, see `llvm_verify`."
+    , "`llvm_points_to`)."
+    , ""
+    , "In order, the arguments to `llvm_compositional_extract` are:"
```

Comment:
As an alternative to just listing numbers (which forces you to manually count the numbers of arguments), it might be worthwhile to give each argument a name in the help text and refer to the argument names in the list below. Something like this, perhaps:

> When invoking `llvm_compositional_extract mod fun_name ...`, the arguments represent the following:
> 
> 1. `mod`: The LLVM module...
> 2. `fun_name`: The name of the function...
> 3. ...